OSXCore/HangulComposer.swift

```diff
@@ -210,6 +221,29 @@ final class HangulComposer: NSObject, Composer {
         let handled = inputContext.process(string.unicodeScalars.first!.value)
         let ucsString = inputContext.commitUCSString
         let recentCommitString = representableString(ucsString: ucsString)
+
+        if configuration.hangulJDKCompatible {
+            if handled {
+                _commitString.append(_compatiblePreeditString)
+                _compatiblePreeditString = ""
+                if recentCommitString.isEmpty {
+                    return .processed
+                } else {
+                    // 마지막 글자가 한글이 아닌 경우 비조합 문자열로 판단한다.
+                    let lastChar = String(Array(recentCommitString).last!)
```

Comment:
```suggestion
                    let lastCharIsAscii = Array(recentCommitString).last!.unicodeScalars.isAscii
```
스칼라 값을 가져오면 아스키인지 확인할수 있어요